Dockerfile

```diff
@@ -43,11 +42,12 @@ RUN helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx && \
 # pulling `terra-app-setup` locally and add cert files to the chart.
 # Leonardo will install the chart from local version.
 # We are also cacheing charts so they are not downloaded with every helm-install
+# If you are changing any versions here, update it in the reference.conf
```

Comment:
how about prefix `WARNING:` cuz it's kinda buried in a blob of text

Reply:
+1